protocol-tokio/Cargo.toml

```diff
@@ -10,10 +10,9 @@ description = """
 Cardano full node network protocol using rust's futures for asynchronous
 network connection support.
 """
-keywords = [ "Cardano", "Protocol" ]
```

Comment:
"blockchain", the hottest keyword in 2019! :)
I think the keywords are canonically lowercase, but it may be unimportant.